backend/fms_core/models/sample.py

```diff
@@ -22,6 +22,7 @@
 from .container import Container
 from .individual import Individual
 from .sample_kind import SampleKind
+from .project import Project
```

Comment:
Is this needed ?

Reply:
Right, @sebastianballesteros this can be removed from the sample model because we don't deal with projects there yet